shared/login/form.js

```diff
@@ -0,0 +1,20 @@
+/* @flow */
+
+import React, {Component} from 'react'
+import Render from './form.render'
+import Intro from './forms/intro'
+import ErrorText from './error.render'
+
+export default class Form extends Component {
+  props: {
+    formComponent: ReactClass
+  };
+
+  render () {
```

Comment:
I agree, I was a bit hesitant because our meta navigator assumes a component. But it really only looks for Component.parseRoute, so an object should be good enough=